Solutions/Azure Activity/Analytic Rules/SubscriptionMigration.yaml

```diff
@@ -0,0 +1,60 @@
+id: 48c026d8-7f36-4a95-9568-6f1420d66e37
+kind: Scheduled
+name: Subscription moved to another tenant
+description: |
+  Detects when a subscription is moved to another tenant. This may indicate a subscription hijacking event.
+severity: High
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+queryPeriod: 10m
+queryFrequency: 5m
```

Comment:
If we query every 5mins for 10mins of data, we are evaluating the same data twice. Either the query should be improved to only check data within last 5mins of ingestion_time or the queryPeriod and queryFrequency should be aligned.

Reply:
This is fair, I was concerned about ingestion delays on the event and that we may miss an event - this may have just been a misconception. If so, I can align the times.